hudi-common/src/main/java/org/apache/hudi/common/fs/SizeAwareFSDataOutputStream.java

```diff
@@ -44,7 +44,7 @@ public class SizeAwareFSDataOutputStream extends FSDataOutputStream {
 
   public SizeAwareFSDataOutputStream(Path path, FSDataOutputStream out, ConsistencyGuard consistencyGuard,
                                      Runnable closeCallback) throws IOException {
-    super(out, null);
+    super(out, null, out.getPos());
```

Comment:
Even for correctness, we should pass the right start position (`out.getPos()`).  Without this, does it overwrite the existing log blocks from the position of 0 when trying to append new log blocks on HDFS?

Reply:
After digging into the code, the start position passed in here is only used by `FSDataOutputStream.PositionCache` to track the current position.  So this is okay.